src/app/resources/resources-add.scss

```diff
@@ -1,48 +1,76 @@
 @import '../variables';
 
 :host {
-  .view-container {
-    display: grid;
-    grid-template-rows: auto 42px;
-    > * {
-      overflow-y: auto;
-    }
-  }
-
-  .actions-container {
-    align-self: center;
-  }
-
-  .form-spacing {
-    width: auto;
-    justify-content: center;
-  }
-
-  .header-label {
-    display: block;
-    margin-top: 10px;
-  }
-
-  .existing-file-container {
-    display: flex;
-    align-items: center;
-  }
-
-  .filename {
-    flex: 1;
-  }
-
-  @media (max-width: $screen-md) {
-    .form-container {
-      width: auto;
-    }
-
-
-    .filename {
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      max-width: 100%;
-    }
-  }
-}
+ .view-container {
+   display: grid;
+   grid-template-rows: auto 42px;
+   width: 100%;
+   > * {
+     overflow-y: auto;
+   }
+ }
+
+ .space-contain {
+   display: flex;
+   flex-direction: column;
+   align-items: flex-start;
+   padding: 0;
+   margin: 0;
+   box-sizing: border-box;
+   width: 100%;
+   min-height: 100vh;
+ }
```

Comment:
By removing the global space-container you end up with a double scrollbar & bypass the styled floating background

![image](https://github.com/user-attachments/assets/4e706556-3f3b-47da-af0b-cf6344aa93c5)